nightly-playground/lib/nightly-playground-stack.ts

```diff
@@ -0,0 +1,57 @@
+/* Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+
+The OpenSearch Contributors require contributions made to
+this file be licensed under the Apache-2.0 license or a
+compatible open source license. */
+
+import { InfraStack } from '@opensearch-project/opensearch-cluster-cdk/lib/infra/infra-stack';
+import { NetworkStack } from '@opensearch-project/opensearch-cluster-cdk/lib/networking/vpc-stack';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class NightlyPlaygroundStack {
+  public stacks: Stack[] = []; // only required for testing purpose
+
+  constructor(scope: Construct, id: string, props: StackProps) {
+    const distVersion = scope.node.tryGetContext('distVersion');
+    if (distVersion === 'undefined') {
+      throw new Error('distVersion parameter cannot be empty! Please provide the OpenSearch distribution version');
+    }
+    const distributionUrl = scope.node.tryGetContext('distributionUrl');
```

Comment:
Can we make the variable naming convention consistent?

Reply:
Same as below